### DISCUSSION TITLE
How active/up to date are these lists?

### DISCUSSION THREAD

**Discussion by epsilonion-liam** [blocklistproject/Lists#1313] [General]
I was wondering if this is still been actively maintained as the lists most recent seen to be like 2 months ago?

I am wondering as I am using these lists and was wondering if you could include the following list.

CSAM - https://github.com/gardenfence/blocklist/blob/main/gardenfence.txt
Its blocking known child abuse URLs and other abusive URLs as its a big thing these days.

**Comment by MNLierman** [comment]
**No – this project is no longer maintained!**
Now, it would be nice if the project maintainers made this clear in the ReadMe.md. There was an update right around the time you posted this comment but it wasn't a very big update, and since then there has been answered comments stating that this is not going to receive any more updates. It seems the project maintainers have no foreseeable interest in maintaining this project. This was not clear to me as I was testing out the block list. Had I known, I would not have even attempted to add or use this block list. 

So for anyone else, **no this is not maintained, and I suggest finding another block list, this one is dead. There is one called Hagezi Pro and Pro++ which are both actively maintained**, feedback is answered, and it receives regular updates. It's not as big as this blocklist, but I don't believe bigger is better. Maintained is better. Blocking a large collection of prehistoric dead domains is not worthwhile or optimal for lookups on a local DNS server. I bet if a script were to ping all of the domains on Blocklist Project, I'm confident at least half would be dead, if not more.

To answer this specific question: you don't need to ask repos to include someone else's list. If you're using something like PiHole or AdGuardHome, you can add multiple lists, and when it updates, it makes a master list combined from all lists you added.

**Reply by AxiomOperator** [reply]
> **No – this project is no longer maintained!** Now, it would be nice if the project maintainers made this clear in the ReadMe.md. There was an update right around the time you posted this comment but it wasn't a very big update, and since then there has been answered comments stating that this is not going to receive any more updates. It seems the project maintainers have no foreseeable interest in maintaining this project. This was not clear to me as I was testing out the block list. Had I known, I would not have even attempted to add or use this block list.
> 
> So for anyone else, **no this is not maintained, and I suggest finding another block list, this one is dead. There is one called Hagezi Pro and Pro++ which are both actively maintained**, feedback is answered, and it receives regular updates. It's not as big as this blocklist, but I don't believe bigger is better. Maintained is better. Blocking a large collection of prehistoric dead domains is not worthwhile or optimal for lookups on a local DNS server. I bet if a script were to ping all of the domains on Blocklist Project, I'm confident at least half would be dead, if not more.
> 
> To answer this specific question: you don't need to ask repos to include someone else's list. If you're using something like PiHole or AdGuardHome, you can add multiple lists, and when it updates, it makes a master list combined from all lists you added.

This is not accurate
